static/app/components/charts/optionSelector.tsx

```diff
@@ -47,6 +47,7 @@ function OptionSelector({
   const mappedOptions = useMemo(() => {
     return options.map(opt => ({
       ...opt,
+      textValue: String(opt.label),
```

Comment:
Would `toString` work?

Reply:
We already use `String(opt.label)` beneath as the text value for `<Truncate />`, so I'm doing the same here.